packages/framework/fluid-framework/README.md

```diff
@@ -2,6 +2,8 @@
 
 The `fluid-framework` package bundles a collection of Fluid Framework client libraries for easy use when paired with a corresponding service client package (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluid-experimental/osdp-client (BETA)`).
 
+This package reexports these APIs from other Fluid client packages: users of this package should not need direct production dependencies on any Fluid packages other than this one and the service client package.
```

Comment:
We should address loggers, dev tools and testing libraries to make it clear if they should be part of this, or apps should import them separately.

Reply:
Done in https://github.com/microsoft/FluidFramework/pull/20955